plutus-benchmark/nofib/test/Spec.hs

```diff
@@ -47,7 +47,7 @@ testClausify = testGroup "clausify"
                , testCase "formula4" $ mkClausifyTest Clausify.F4
                , testCase "formula5" $ mkClausifyTest Clausify.F5
                , runTestNested $
-                  Tx.goldenSize "formula1 (size)" (Clausify.mkClausifyCode Clausify.F1)
+                  Tx.goldenSize "formula1-size" (Clausify.mkClausifyCode Clausify.F1)
```

Comment:
the golden files have the suffix `uplc-size`, so I can take out the "size" here if you like. Actually yeah I'll do that.

Reply:
Yeah, I was going to suggest that but then the test names are the same as other tests, which does show up in the test output from `tasty`. IDK if it makes anything not work, but it seems maybe bad.

Reply:
like, if you wanted to rerun just that one test it might be a problem, but that's not such a big deal

Reply:
hmmm....yeah. I can add the `-size` back in then? or maybe add a `-golden` in?